### ISSUE TITLE
Sometimes cant login, other times just won't download torrents (IPT)

### ISSUE THREAD

**Issue by jrochet** [CouchPotato/CouchPotatoServer#6831]
### Steps to reproduce:
1. .. Add movie.
2. ..... or just click refresh on an already-added movie.

### Information:
Movie(s) I have this with: ... All movies
Quality of the movie being searched: ... Best
Providers I use: ...IPT
Version of CouchPotato: desktop: 3.0.1
Running on: ... Windows 10

### Logs:
```
09-04 15:18:53  INFO [  couchpotato.core.loader] Loaded media_movie_providers_userscript: moviemeter 
09-04 15:18:53  INFO [  couchpotato.core.loader] Loaded media_movie_providers_userscript: moviesio 
09-04 15:18:53  INFO [  couchpotato.core.loader] Loaded media_movie_providers_userscript: reddit 
09-04 15:18:53  INFO [  couchpotato.core.loader] Loaded media_movie_providers_userscript: rottentomatoes 
09-04 15:18:53  INFO [  couchpotato.core.loader] Loaded media_movie_providers_userscript: tmdb 
09-04 15:18:53  INFO [  couchpotato.core.loader] Loaded media_movie_providers_userscript: trakt 
09-04 15:18:53  INFO [  couchpotato.core.loader] Loaded media_movie_providers_userscript: youteather 
09-04 15:18:53  INFO [  couchpotato.core.loader] Loaded media_movie: searcher 
09-04 15:18:53  INFO [  couchpotato.core.loader] Loaded media_movie: suggestion 
09-04 15:18:54  INFO [       couchpotato.runner] Starting server on port 8083 
09-04 15:18:54  INFO [chpotato.core._base._core] Launching browser 
09-04 15:18:54  INFO [tato.core._base.scheduler] Scheduling "movie.searcher.all", cron: day = *, hour = 0, minute = 50 
09-04 15:18:54  INFO [hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/configuration?api_key=xxx 
09-04 15:18:54  INFO [tato.core._base.scheduler] Scheduling renamer.check_snatched, interval: hours = 0, minutes = 1, seconds = 0 
09-04 15:18:54  INFO [tato.core._base.scheduler] Scheduling updater.check, interval: hours = 6, minutes = 0, seconds = 0 
09-04 15:18:54  INFO [tato.core._base.scheduler] Scheduling renamer.check_snatched_forced, interval: hours = 2, minutes = 0, seconds = 0 
09-04 15:18:54  INFO [otato.core.plugins.manage] Updating manage library: Z:\Media\Movies 
09-04 15:18:54  INFO [hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/messages/?last_check=1384713000, data: [] 
09-04 15:18:54  INFO [tato.core._base.scheduler] Scheduling automation.add_movies, interval: hours = 12, minutes = 0, seconds = 0 
09-04 15:18:54  INFO [potato.core._base.updater] === VERSION desktop: 3.0.1, using DesktopUpdater === 
09-04 15:19:09  INFO [hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/ismovie/tt2395427/?adding=1, data: [] 
09-04 15:19:09  INFO [hpotato.core.plugins.base] Opening url: get https://api.couchpota.to/info/tt2395427/, data: [] 
09-04 15:19:09  INFO [hpotato.core.plugins.base] Opening url: get https://api.themoviedb.org/3/movie/tt2395427?api_key=xxx&append_to_response=alternative_titles, data: [] 
09-04 15:19:09  INFO [hpotato.core.plugins.base] Opening url: get http://www.omdbapi.com/?type=movie&i=tt2395427, data: [] 
09-04 15:19:09  INFO [ie.providers.info.omdbapi] Found: Avengers: Age of Ultron (2015) 
09-04 15:19:09  INFO [ie.providers.info.omdbapi] Found: Avengers: Age of Ultron (2015) 
09-04 15:19:09  INFO [hpotato.core.plugins.base] Opening url: post https://api.couchpota.to/suggest/, data: ['ignore', 'movies'] 
```Paste the text here

**Comment by CouchPotatoBot** [comment]
Thanks for the issue report! Before a real human comes by, please make sure your report has all the below criteria checked

- [ ] Fill in all the dots. They're there for a reason
- [x] Post logs, either inline (for smaller logs) or using [Pastebin](http://pastebin.com/)
- [x] Please wrap inline logs inside ``` for readability
- [ ] Enabled debug logging in Settings > Advanced (restart CP, be sure to disable after the bug is fixed)
- [x] Post full version information

Please make sure you also read [contribution guide](https://github.com/RuudBurger/CouchPotatoServer/blob/develop/contributing.md#issues) and followed all the steps. 
Make the title describe your issue. Having "CP not working" or "I get this bug" for 100 issues, isn't really helpful. My master will close issues if there isn't enough information. On a good day he will tag the issue on close with the reason (like `can't reproduce`), but usually he won't, the lazy asshat.

Sometimes my master seems like a grumpy cat and responds with short answers. This isn't (always) because he hates you, but because he's on mobile or busy fixing bugs. If something isn't clear, please let him know. Maybe he can teach me his awesome ways.

Thanks!

**Comment by jrochet** [comment]
I am closing this issue as it is resolved.  Here is the resolution:

I went away from using the Windows installer version (3.0.1) and instead used the run from source method which was outlined pretty nicely by you guys.  That version info is below (current as of Sept 6th it appears), and it works like a charm.  I installed it as a service with nssm and that also works great. 


Version: 4a0999d9d2a323a4005e88bffb47e6918ae51e14 (06/09/2016, 10:53:22)
Updater: source, master
ID: 11100

Thanks very much, I was being impatient as well and so I apologize.
